### PR TITLE
Implement MSP_STATUS_EX to display CPU load

### PR DESCRIPTION
**Pull request by gaelj**
Addresses https://github.com/betaflight/betaflight-configurator/issues/82
Requires https://github.com/betaflight/betaflight/pull/776

### CODE REVIEW

**Comment by borisbstyle** [comment]
I think we miss the version checks? Or are those already in place for this command?

File: main.html [hunk 1]
```diff
@@ -250,6 +250,9 @@
             <div>
                 <span i18n="statusbar_cycle_time"></span> <span class="cycle-time">0</span>
             </div>
+            <div>
+                <span class="cpu-load">CPU Load: 0%</span>
```
Comment (mikeller) [inline main.html]: I think this should be using `i18n="statusbar_cpu_load"` that you are defining in `messages.json` above.

File: js/serial_backend.js [hunk 2]
```diff
@@ -250,7 +250,7 @@ function onConnect() {
     $('#tabs ul.mode-disconnected').hide();
     $('#tabs ul.mode-connected').show(); 
      
-    MSP.send_message(MSP_codes.MSP_STATUS, false, false);
```
Comment (mikeller) [inline js/serial_backend.js]: This has to be version gated to not break non-3.0 users.

**Comment by borisbstyle** [comment]
Yes and it should be hidden for non 3.0 users

**Comment by gaelj** [comment]
Oh right. I will correct asap

**Comment by gaelj** [comment]
@borisbstyle @mikeller ; corrected.

**Comment by borisbstyle** [comment]
Cool! lets merge it

**Pull request merged**